### PR TITLE
doc: Update versions for 2.2.0 release

### PR DESCRIPTION
**Pull request by umapraseeda**
Update changelog versions for 2.2.0 release

Signed-off-by: UMA PRASEEDA <uma.praseeda@nordicsemi.no>

### CODE REVIEW

**Review by peknis — APPROVED** [review]

File: gzll/CHANGELOG.rst [hunk 1]
```diff
@@ -9,10 +9,12 @@ Changelog
...
 
-Main branch
-***********
+nRF Connect SDK v2.2.0
+**********************
+
+All the notable changes included in the |NCS| v2.2.0 release are documented in this section.
 
-Support nRF5340 network core.
+* Support nRF5340 network core.
```
Comment (leewkb4567) [inline gzll/CHANGELOG.rst:17]: Suggest removing this 'support nRF534' line instead. The style is thus more consistent with the other nrfxlib changelogs.

Comment (umapraseeda) [reply gzll/CHANGELOG.rst]: fixed

**Review by rugeGerritsen — APPROVED** [review]

**Review by leewkb4567 — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

File: lc3/CHANGELOG.rst [hunk 2]
```diff
@@ -5,8 +5,10 @@ Changelog
 
 All notable changes to this project are documented in this section.
```
Comment (greg-fer) [inline lc3/CHANGELOG.rst:6]:
```suggestion
All notable changes to this project are documented on this page.
```

File: lc3/CHANGELOG.rst [hunk 3]
```diff
@@ -5,8 +5,10 @@ Changelog
...
 
-Main branch
-***********
+nRF Connect SDK v2.2.0
+**********************
+
+All the notable changes included in the |NCS| v2.2.0 release are documented in this section.
```
Comment (greg-fer) [inline lc3/CHANGELOG.rst:11]:
```suggestion
Here are all the notable changes included in the |NCS| v2.2.0 release:
```

File: nrf_dm/doc/CHANGELOG.rst [hunk 4]
```diff
@@ -9,18 +9,18 @@ Changelog
 
 All the notable changes to this project are documented in this file.
```
Comment (greg-fer) [inline nrf_dm/doc/CHANGELOG.rst:10]:
```suggestion
All the notable changes to this project are documented on this page.
```

File: gzll/CHANGELOG.rst [hunk 5]
```diff
@@ -9,10 +9,10 @@ Changelog
 
 All notable changes to this project are documented in this file.
```
Comment (greg-fer) [inline gzll/CHANGELOG.rst:10]:
```suggestion
All the notable changes to this project are documented on this page.
```

File: nrf_rpc/CHANGELOG.rst [hunk 6]
```diff
@@ -9,10 +9,10 @@ Changelog
 
 All notable changes to this project in the |NCS| are documented in this file.
```
Comment (greg-fer) [inline nrf_rpc/CHANGELOG.rst:10]:
```suggestion
All the notable changes to this project are documented on this page.
```

File: softdevice_controller/CHANGELOG.rst [hunk 7]
```diff
@@ -9,10 +9,10 @@ Changelog
 
 All the notable changes to this project are documented in this file.
```
Comment (greg-fer) [inline softdevice_controller/CHANGELOG.rst:10]:
```suggestion
All the notable changes to this project are documented on this page.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Some consistency edits.

**Comment by umapraseeda** [comment]
@greg-fer all suggestions applied. Thanks!

**Review by greg-fer — APPROVED** [review]

**Comment by nordicjm** [comment]
@b-gent @koffes @alexsven @Tschet1 @eriksandgren @doki-nordic @KAGA164 can you please check and approve this?

**Pull request merged**